pkgs/top-level/python-packages.nix

```diff
@@ -20162,6 +20161,22 @@ in {
     };
   };
 
+  pyext = buildPythonPackage rec {
+    name = "pyext-${version}";
+    version = "0.7";
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/b0/be/9b6005ac644aaef022527ce49617263379e49dbdbd433d1d3dd66d71f570/pyext-0.7.tar.gz";
```

Comment:
and can you put this new package in a new file in `pkgs/development/python-modules/pyext/default.nix`